app/Enums/PaymentMethod.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace App\Enums;
+
+enum PaymentMethod : int
+{
+    case CARD = 1;
+    case GCASH = 2;
+    case PAYMAYA = 3;
+    case GRABPAY = 4;
+    case OTC = 5;
+
+    /**
+     * Shows the proper label after casting
+     */
+    public function label() : string
+    {
+        return match($this)
+        {
+            self::CARD => 'Credit/Debit Card',
+            self::GCASH => 'GCash',
+            self::PAYMAYA => 'PayMaya',
+            self::GRABPAY => 'GrabPay',
+            self::OTC => 'Over the Counter',
+        };
+    }
+
+    /**
+     * Populate select options for categories
+     * 
+     * @return array
+     */
+    static function getOptions() {
+        $items = [];
+
+        foreach (self::cases() as $case) {
+            $items[$case->value] = $case->label();
+        }
+
+        return $items;
+    }
```

Comment:
Potentially duplicated code. This can also be found in another enum, so it's best if we can place code like this in a trait and call it for classes that have need of it.